From 302c5dc6f92d4e8502e47672f89494deb332cb7b Mon Sep 17 00:00:00 2001
From: Jacob Weinstock <jakobweinstock@gmail.com>
Date: Sun, 5 Mar 2023 09:24:47 -0700
Subject: [PATCH] Add nix location for isolinux.bin

Signed-off-by: Jacob Weinstock <jakobweinstock@gmail.com>
---
 src/util/genfsimg | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/util/genfsimg b/src/util/genfsimg
index 0c069279..a40fd520 100755
--- a/src/util/genfsimg
+++ b/src/util/genfsimg
@@ -107,6 +107,7 @@ copy_syslinux_file() {
 	/usr/local/share/syslinux/bios/com32/elflink/ldlinux \
 	/usr/local/share/syslinux/modules/bios \
 	/usr/lib/ISOLINUX \
+	$(echo $buildInputs | tr ' ' '\n' | grep syslinux)/share/syslinux \
 	; do
 	if [ -e "${SRCDIR}/${FILENAME}" ] ; then
 	    install -m 644 "${SRCDIR}/${FILENAME}" "${DESTDIR}/"
-- 
2.34.1

